deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -235,10 +235,33 @@ def delete_file(file_path):
 
 
 def delete_path_recursively(path):
+    '''Remove a directory recursively.
+
+    Args:
+        path (str): Directory path.
+    '''
     if os.path.exists(path):
         shutil.rmtree(path, onerror=on_write_error)
 
 
+def remove_files_in_folder(dir_path):
+    '''Remove files (and directories) recursively preserving the directory.
+
+    Args:
+        dir_path (str): The path of the directory that contains all the files to be removed.
+    '''
+    files_in_path = os.listdir(dir_path)
+    for filename in files_in_path:
+        file_path = os.path.join(dir_path, filename)
+        try:
+            if os.path.isfile(file_path) or os.path.islink(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path):
+                delete_path_recursively(file_path)
+        except (OSError, PermissionError) as error:
+            print(f"Failed to remove {file_path}.\nError: {error}")
```

Comment:
Don't catch the exception. You must to raise it

Reply:
Done in [8a48b86](https://github.com/wazuh/wazuh-qa/pull/3837/commits/8a48b86915fca9d798ad4f96f2972b288fbf414f)